code/atelier style.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+body {
+background: whitesmoke;
+color: black;
+margin: 0;
+font-family: 'Montserrat', sans-serif;
+}
+
+h1 {
+  display: flex;
+  font-size: 40px;
+  color: white;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 32px;
+}
+
+h3 {
+  font-size: 26px;
+}
+
+h4 {
+  font-size: 32px;
+  font-weight: 100;
+  margin-top: -16px;
+  margin-bottom: 16px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1400px;
+  margin: 0 auto;
+}
+
+/*** HEADER ***/
+
+header {
+  background: #24292d;
+  height: 80px; 
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+
+}
+
+.header-container {
+  padding: 100px 16px;
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.header-container div {
+  width: 30px;
+  height: 5px;
+  background: whitesmoke;
+  text-align: center;
+}
+
+nav {
+display: none;
+}
+
+nav a {
+  color: white;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+/*** WELCOME CONTAINER ***/
+
+.welcome-container .container {
+  padding: 50px 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+.welcome-container img {
+  width: 100%;
+}
+
+.welcome-container div {
+  text-align: left;
+}
+
+/*** CHALLENGES SECTION ***/
+
+.challenges-container {
+  background: #fff;
+  color: black;
+  text-align: center;
+  padding: 24px;
+}
+
+.challenges {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.challenges a {
+  background: whitesmoke;
+  border: 1.5px solid #24292d;
+  border-radius: 16px;
+  width: 100%;
+  text-decoration: none;
+  color: #000;
+  box-shadow: 12px 12px 0 0 #d8eee6;
+  padding: 5px;
+  margin-bottom: 32px;
+}
+
+.challenges a img {
+  width: 80%;
+  margin: 20px;
+}
+
+.circles {
+  display: flex;
+  justify-content: space-between; 
+  margin: 16px;
+  margin-bottom: -16px;
+}
+
+.circles span {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: teal;
+}
+
+.circles span:last child {
+  background: #5583c3;
+}
+
+/*** FOOTER ***/
+
+.footer {
+  padding-left: 16px;
+  }
+
+/*** MEDIA QUERIES ***/
+
+@media (min-width: 650px) {
+  
+  header {
+    justify-content: space-between;
+  }
+
+  nav {
+    display: inline-block;
+  }
+
+  .header-container {
+    display: none;
+  }
+
+  .welcome-container div {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .welcome-container img {
+    width: 100%;
+  }
+
+  .challenges a img {
+    width: 300px;
+    height: 300px;
+    margin: 20px;
+  }
```

Comment:
These on the other hand - you want to keep dynamic because all of a sudden 300px is to wide for the card, see image. Add object-fit: cover; to keep the aspect ratio:
    ```
width: 80%;
    height: 50%;
    object-fit: cover;
```
![Skärmavbild 2020-10-28 kl  14 41 07](https://user-images.githubusercontent.com/51529201/97443678-9e984700-192b-11eb-9a02-1ceb8349f6b7.png)